src/chains/index.ts

```diff
@@ -0,0 +1,18 @@
+import { RpcUrl } from "../types";
+
+const _getMainnetRPC = (name: string): RpcUrl =>
+  `https://mainnet.skalenodes.com/v1/${name}`;
+const _getTestnetRPC = (name: string): RpcUrl =>
+  `https://staging-v3.skalenodes.com/v1/${name}`;
+
+export const chains: { [key: string]: RpcUrl } = {
```

Comment:
Nice, I like it so we can access to like chains.calypso or chains["calypso-testnet"]

Let's add 2 more objects inside like mainnet and testnet and add inside hub names so we would be able to access it like chains.mainnet.calypso or chains.testnet.chaos - I think it would be more convenient.

Would like to here your thoughts:)

Reply:
The reason I did the way I did was to make it as simple as possible on the developer side. 
So when creating request instead of requiring two additional params i.e 

```typescript

/** What you are proposing */
function callFunction(name: string, network: "mainnet" | "testnet") {}
callFunction("calypso", "testnet")

/** What I am proposing */
function callFunction(name: string) {}
callFunction("calypso") -> resolves to mainnet rpc
// or
callFunction("calypso-testnet") -> resolves to testnet rpc